docs/user_guide/text_parser.md

```diff
@@ -0,0 +1,178 @@
+# text_paser
```

Comment:
I wonder if it would be worth linking to the module's source, so they can see the other options available, such as `dir`?